lib/pbench/server/database/models/auth_tokens.py

```diff
@@ -45,7 +48,9 @@ def delete(auth_token: str):
             auth_token : the auth token to delete
         """
         try:
-            Database.db_session.query(ActiveTokens).filter_by(token=auth_token).delete()
+            Database.db_session.query(AuthToken).filter_by(
+                auth_token=auth_token
+            ).delete()
```

Comment:
Can we do this in two, nice (non-wrapped) lines instead of the ugly three which Black provides for us?  E.g.,
```
            t = Database.db_session.query(AuthToken).filter_by(auth_token=auth_token)
            t.delete()
```
or
```
            tokens = Database.db_session.query(AuthToken)
            tokens.filter_by(auth_token=auth_token).delete()
```

Reply:
I'd rather give Python the information that we don't need the intermediate over code formatting.

Future changes are going to drop the long `Database.db_session`, which will help.

Perhaps instead:

```
_q = Database.db_session.query
_q(AuthToken).filter_by(auth_token=auth_token).delete()
```

Reply:
> I'd rather give Python the information that we don't need the intermediate over code formatting.

Gosh, I would hope that Python can figure that out on its own, given that there aren't any further references to the variable in-scope.  (And, I would hope that there's an optimization step which gets rid of it, regardless of how we construct the source!)

> Perhaps instead

Sure, that would be fine, but there's no reason to include the leading underscore in the variable name -- the variable is already limited in scope, and it's obvious from context that it's local, and the leading underscore is just an ugly waste of space.

Reply:
What is it with leading underscores?  I really feel bad for them.

Reply:
Yeah, I know, they never get to go first. 🤣

(But, even though _my_ name doesn't start with an underscore, I didn't get to go first either....)

Reply:
Fixed.